CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ Changelog for Razorpay-PHP SDK. Follows [keepachangelog.com](https://keepachange
 
 ## Unreleased
 
+
+## [2.8.6] - 2022-12-12
+
+- [#336](https://github.com/razorpay/razorpay-php/pull/336) [`7631f52`](https://github.com/razorpay/razorpay-php/commit/7631f52a98f88496b733ba6e741ded649d9b95e4) : Thanks [@ankitdas13](https://github.com/ankitdas13)! - fix: response typecast issue
```

Comment:
Can we add a little more detail here? What did we fix? How does it affect users? etc.

Reply:
@sidag95 Object property in response was shown incorrect type so we fixed that and it won't affect users

ex : 

```Razorpay\Api\Payment Object
(
[attriburtes:protected]: Array
  (
   [id] => pay_XXXXXXXXXX
   [notes] => Razorpay\Api\Payment Object      Replaced this type to Std class
     (
        [attriburtes:protected]: Array
        (
           [woocommerce_order_id] => 1055128
        )
     )
  )
```
to

```Razorpay\Api\Payment Object
(
[attriburtes:protected]: Array
  (
   [id] => pay_XXXXXXXXXX
   [notes] => Razorpay\Api\StdClass Object
     (
        [attriburtes:protected]: Array
        (
           [woocommerce_order_id] => 1055128
        )
     )
  )
```